### PR TITLE
docs: Add announcement for new repository URLs

### PR DESCRIPTION
**Pull request by spencergilbert**

### CODE REVIEW

**Comment by spencergilbert** [comment]
@bkabrda if you'd like to review anything here now that you're back.

**Review by tobz — APPROVED** [review]

File: website/content/en/highlights/2023-11-07-new-linux-repos.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+curl https://keys.datadoghq.com/DATADOG_APT_KEY_CURRENT.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+curl https://keys.datadoghq.com/DATADOG_APT_KEY_F14F620E.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+curl https://keys.datadoghq.com/DATADOG_APT_KEY_C0962C7D.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+```
+
+4. Run the following commands to update your local `apt` repo and install Vector:
+
+```sh
+sudo apt-get update
+sudo apt-get install vector datadog-signing-keys
```
Comment (bkabrda) [inline website/content/en/highlights/2023-11-07-new-linux-repos.md:74]: I think right now we're not distributing the datadog-signing-keys package through the vector repos, so let's remove it until we do. We'll have to make some minor modification in it to support vector.

Comment (spencergilbert) [reply website/content/en/highlights/2023-11-07-new-linux-repos.md]: Addressed in 9b67ba9c6

**Review by bkabrda — APPROVED** [review]

**Review by maycmlee — COMMENTED** [review]
Left some docs suggestions! Let me know if you have any questions or if I changed the meaning of anything.

File: website/content/en/highlights/2023-11-07-new-linux-repos.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+new repository.
+
+```sh
+CSM_MIGRATE=true bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_vector0.sh)"
+```
+
+Alternatively, `CSM_MIGRATE` may be left unset to leave the removal of the
+existing repository to your discretion.
+
+### Manual step by step instructions
```
[the remark this answers is not in the document]
Comment (spencergilbert) [reply website/content/en/highlights/2023-11-07-new-linux-repos.md]: 👍 FWIW, I think the Datadog Agent docs (in-app) uses the same hyphen-less style. Not sure if we want it changed there as well.

**Comment by spencergilbert** [comment]
Thanks @maycmlee!

**Review by maycmlee — APPROVED** [review]

File: website/content/en/highlights/2023-11-07-new-linux-repos.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+If you have any questions or concerns don't hesitate to reach out on [Discord]
+or [Discussions]!
+
+## Migration guide
+
+The following command **removes** the existing repository and configures the
+new repository.
+
+```sh
+CSM_MIGRATE=true bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_vector0.sh)"
```
Comment (jszwedko) [inline website/content/en/highlights/2023-11-07-new-linux-repos.md:35]: When running this script for the RPM installation, it prompted me when running `yum install vector` to accept the new keys. Is that expected? I somewhat expected this script to trust the keys already.

**Review by jszwedko — COMMENTED** [review]
Great!

The docs looked good. I tested different flows for apt and rpm:

* Just using the new install script
* Using the old install script, followed by the new one without migration
* Using the old install script, followed by the new one with migration
* Using the manual instructions

One question I have is about the script naming. The current naming of `install_script_vector0.sh` makes it sound like it is actually going to install Vector, but it only sets up the repos. I'm wondering if a better name might be something like `setup_repos_vector0.sh`?

**Comment by spencergilbert** [comment]
> One question I have is about the script naming. The current naming of install_script_vector0.sh makes it sound like it is actually going to install Vector, but it only sets up the repos. I'm wondering if a better name might be something like setup_repos_vector0.sh?

I did consider this, but ultimately felt keeping it consistent with the other scripts felt _right_, especially considering the possibility of replacing the existing "install" script with one generated from the DD template - reducing maintenance/drift.

Of course that was just theoretical, and if we'd rather be more accurate today and change it down the line (if we even go that way) we could rename it.

Alternatively, @bkabrda had mentioned you had spoken and mentioned wanting the script available under the `vector.dev` domain - and the current location _isn't_. We _could_ bypass the user side naming and add a redirect from `setup.vector.dev` (or whatever) to the S3 URL, that way internally the script is consistent but the user doesn't see it.

**Comment by jszwedko** [comment]
> > One question I have is about the script naming. The current naming of install_script_vector0.sh makes it sound like it is actually going to install Vector, but it only sets up the repos. I'm wondering if a better name might be something like setup_repos_vector0.sh?
> 
> I did consider this, but ultimately felt keeping it consistent with the other scripts felt _right_, especially considering the possibility of replacing the existing "install" script with one generated from the DD template - reducing maintenance/drift.
>
> Of course that was just theoretical, and if we'd rather be more accurate today and change it down the line (if we even go that way) we could rename it.

Ah, yeah, that is fair. I think the Datadog Agent install scripts go further and actually _install_ the Agent though, right? It seems like we wouldn't be able to template this until we unify the behavior and have the script instal Vector as well. The current scripts 

> Alternatively, @bkabrda had mentioned you had spoken and mentioned wanting the script available under the `vector.dev` domain - and the current location _isn't_. We _could_ bypass the user side naming and add a redirect from `setup.vector.dev` (or whatever) to the S3 URL, that way internally the script is consistent but the user doesn't see it.

Also true, we did discuss hosting it on the Vector domain. We could do that via a [redirect](https://github.com/vectordotdev/vector/blob/master/netlify.toml) right now. I like `setup.vector.dev`. Maybe we can just go that route and leave the S3 name the same?

**Comment by spencergilbert** [comment]
> Ah, yeah, that is fair. I think the Datadog Agent install scripts go further and actually _install_ the Agent though, right? It seems like we wouldn't be able to template this until we unify the behavior and have the script instal Vector as well. The current scripts

Sorry, unclear from my end - the DD (agent/opw) scripts do go further. As-is we could template it from _just_ the repo setup snippets.

> Also true, we did discuss hosting it on the Vector domain. We could do that via a [redirect](https://github.com/vectordotdev/vector/blob/master/netlify.toml) right now. I like `setup.vector.dev`. Maybe we can just go that route and leave the S3 name the same?

Yep, that was my suggestion as well - of course if they look at the redirect they'll see the name which may be confusing, but the bucket name may also be confusing as well 🤷.

**Comment by jszwedko** [comment]
I updated the dates to reflect the new plan and added a redirect for `setup.vector.dev`.

**Pull request merged**